packaging/windows/host/host.wxs

```diff
@@ -35,9 +34,13 @@
         </Directory>
     </Fragment>
     <Fragment>
-        <ComponentGroup Id="InstallFiles">
+        <ComponentGroup Id="InstallSharedHostandDetectionKeys">
```

Comment:
For step 4, the following regkey is removed. (taking x86 as an example) 
     HKEY_LOCAL_MACHINE\SOFTWARE\dotnet\Setup\InstalledVersions\x86\sharedhost

The hostfxr and shared framework regkeys are not impacted.